app/webpacker/components/CompetitionsOverview/filterUtils.js

```diff
@@ -89,13 +82,13 @@ export const updateSearchParams = (searchParams, filterState, displayMode) => {
 
   // for date values, format and add them if applicable, otherwise omit them
```

Comment:
comment needs updating since the code no longer formats

Reply:
Well, it kinda does but just in a different place :smile: (The `UtcDatePicker` component internally does all the formatting)

Good catch!